### ISSUE TITLE
Support lines history

### ISSUE THREAD

**Issue by NicolasLM** [rr-/urwid_readline#14]
One of the features of GNU Readline we are all familiar with is the history: `up` for the previous command, `ctrl r` for reverse search...

It would be a great addition if urwid_readline supported similar shortcuts.

**Comment by gs-niteesh** [comment]
Is there any idea to support this feature? Atleast the 'up' for previous command?

**Comment by qhuy4119** [comment]
I don't think this functionality is `urwid_readline`'s job. What is a command? urwid_readline provides a `ReadlineEdit` widget where you can type text, that's it. In `ReadlineEdit`, if you press `Enter`, it sets `self.multiline=True` and insert a `\n` without executing anything.
